pxtblocks/blocklyloader.ts

```diff
@@ -119,15 +124,25 @@ namespace pxt.blocks {
             let attr = attrNames["this"];
             block.appendChild(createShadowValue(attr.name, attr.type, attr.shadowValue || attr.name, attr.shadowType || "variables_get"));
         }
-        if (fn.parameters)
+        if (fn.parameters) {
             fn.parameters.filter(pr => !!attrNames[pr.name].name &&
                 (/^(string|number|boolean)$/.test(attrNames[pr.name].type)
                     || !!attrNames[pr.name].shadowType
                     || !!attrNames[pr.name].shadowValue))
                 .forEach(pr => {
                     let attr = attrNames[pr.name];
-                    block.appendChild(createShadowValue(attr.name, attr.type, attr.shadowValue, attr.shadowType));
+                    let shadowValue = createShadowValue(attr.name, attr.type, attr.shadowValue, attr.shadowType);
+                    if (fn.attributes.min && fn.attributes.max) {
+                        if (fn.attributes.min[pr.name] && fn.attributes.max[pr.name]) {
+                            let container = document.createElement('mutation');
+                            container.setAttribute('min', fn.attributes.min[pr.name]);
+                            container.setAttribute('max', fn.attributes.max[pr.name]);
+                            shadowValue.firstChild.appendChild(container);
```

Comment:
The problem with that solution was that when I do the regex to extract Min and Max I don't have access to the parameters, I just know their names and description.